packages/superset-ui-core/src/query/constants.ts

```diff
@@ -17,3 +25,44 @@
  * under the License.
  */
 export const DTTM_ALIAS = '__timestamp';
+
+export const EXTRA_FORM_DATA_OVERRIDE_REGULAR_KEYS: (keyof ExtraFormDataOverrideRegular)[] = [
+  'granularity',
+  'granularity_sqla',
+  'time_column',
+  'time_grain',
+  'time_range',
+];
```

Comment:
better do it as I think as:
```
export const EXTRA_FORM_DATA_OVERRIDE_REGULAR_KEYS: (keyof ExtraFormDataOverrideRegular)[] = Object.values(EXTRA_FORM_DATA_OVERRIDE_REGULAR_MAPPINGS);
```

Reply:
Good idea 👍  I assume you mean `Object.keys`.

Reply:
I still think `values` because in the end we use `values`

Reply:
We don't use the values in `ExtraFormData` - AFAIK `Object.values` is actually `keyof QueryObject` as stated in the `EXTRA_FORM_DATA_OVERRIDE_REGULAR_MAPPINGS` declaration.